Services/WebServices/ECS/classes/class.ilAuthProviderECS.php

```diff
@@ -201,33 +215,7 @@ public function validateHash()
 			
 			$this->getLogger()->dump($auths, ilLogLevel::DEBUG);
 
-			if($auths->pid)
-			{
-				try
-				{
-					include_once './Services/WebServices/ECS/classes/class.ilECSCommunityReader.php';
-					$reader = ilECSCommunityReader::getInstanceByServerId($this->getCurrentServer()->getServerId());
-					$part = $reader->getParticipantByMID($auths->pid);
-					
-					if(is_object($part) and is_object($part->getOrganisation()))
-					{
-						$this->abreviation = $part->getOrganisation()->getAbbreviation();
-					}
-					else
-					{
-						$this->abreviation = $auths->abbr;
-					}
-				}
-				catch(Exception $e)
-				{
-					$this->getLogger()->warning('Authentication failed with message: ' . $e->getMessage());
-					return false;
-				}
-			}
-			else
-			{
-				$this->abreviation = $auths->abbr;
-			}
+			$this->abreviation = $auths->abbr;
```

Comment:
Unfortunately $auths->abbr is marked as deprecated in https://www.freeit.de/documents/ecsa/ecsa.pdf chapter 3.7.3
So we cannot trust this value.